build/build.sh

```diff
@@ -48,26 +47,21 @@ if [[ "${REVISION}" == "${LAST_REVISION}" ]]; then
   exit
 fi
 
-export LLVM_HOME=${ROOT}/llvm
-export ISPC_HOME=${ROOT}/ispc
+ISPC_HOME=${ROOT}/ispc
 
 # Checkout
 git clone --depth 1 --single-branch -b "${BRANCH}" "${URL}" "${ISPC_HOME}"
 
-cd "${ISPC_HOME}"
```

Comment:
Without this where does the LLVM come from?

Reply:
LLVM comes from `LLVM_URL` and `LLVM_VERSION` defined here https://github.com/ispc/ispc/blob/main/superbuild/osPresets.json